<?xml version="1.0"?>
<launch>

    <!-- <include file="$(find usb_cam)/launch/usb_cam-test.launch"/> -->
    
    <arg name="video_format" default=".mp4"/>
    <arg name="output_prepend" default=""/>
    <arg name="input_stream" default="/usb_cam/image_raw"/>
    


    <node name="image_stream_encoder" pkg="image_stream_encoder" type="image_stream_encoder_node" output="screen">
        <param name="video_format" value="$(arg video_format)"/>
        <param name="output_prepend" value="$(arg output_prepend)"/>
        <param name="input_stream" value="$(arg input_stream)"/>
    </node>
    

</launch>